tutorial.rst

```diff
@@ -194,7 +199,9 @@ Please read the comments in the code to understand what is needed for the projec
             # Initialize the IPM library with a reference to the forward kinematics of the robot
             # We also need to provide the camera info, this is optional during the initialization
             # as it can be provided via a setter later on as well
-            self.ipm = IPM(self.tf_buffer, self.camera_info)
+            # We can also set the distortion parameter to True if we want to use the distortion model 
+            # from the camera info and don't want to rectify the image beforehand
+            self.ipm = IPM(self.tf_buffer, self.camera_info, distortion=True)
```

Comment:
It is True

Reply:
I agree that the wording is a bit weird, but I don't think the comment necessarily contradicts the fact that it is true in the example.